Gemfile

```diff
@@ -1,6 +1,6 @@
-source 'http://rubygems.org'
+source 'https://rubygems.org'
```

Comment:
Hey Yes!! I'm thinking about releasing a new gem version with the PRs that have been merged and since yours it's a bit longer have a separate version for this one.  I didn't see any blockers but I want to figure out if it makes sense to release #59 now as well.  

Thanks for your help again and I'll ping you if I need something from your end.  👍